app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    name_file = command.split(" ")
```

Comment:
The variable `name_file` is expected to have at least three elements (command, source file, destination file). Consider adding error handling to check if the command is properly formatted and contains the expected number of elements.